runtime/compiler/runtime/JITServerAOTSerializationRecords.hpp

```diff
@@ -286,6 +288,36 @@ struct AOTHeaderSerializationRecord : public AOTSerializationRecord
    };
 
 
+struct ThunkSerializationRecord : public AOTSerializationRecord
+{
+public:
+   size_t signatureSize() const { return _signatureSize; }
+   size_t thunkCodeSize() const { return _thunkCodeSize; }
+   const uint8_t *signature() const { return _varSizedData; }
+   const uint8_t *thunkCode() const { return _varSizedData + _signatureSize; }
```

Comment:
I was thinking about this as well, but I reached the conclusion that alignment may not matter here, since we are not executing the code directly. The alignment matters when the thunk is copied into the code cache.